testplan/common/utils/validation.py

```diff
@@ -30,3 +30,7 @@ def _validator(kls):
 def is_valid_url(url):
     """Validator that checks if a url is valid"""
     return True if validators.url(url) is True else False
+
+
+def is_valid_email(email):
+    return True if validators.email(email) is True else False
```

Comment:
why not simply `return validators.email(email)`...?

Reply:
Because the function returns an Exception object if validation fails, sound a little weird but it does in that way

Reply:
Oh right, in that case I think you can convert to bool like `return bool(validators.email(email)` - from the docs, validation failures are Falsey: https://validators.readthedocs.io/en/latest/#basic-validators